app/client/src/components/pages/StateTribal.Routes.StateTribalTabs.js

```diff
@@ -1,8 +1,7 @@
 // @flow
 
-import React, { useContext, useEffect, useRef } from 'react';
+import { useContext, useEffect, useRef } from 'react';
```

Comment:
```suggestion
/** @jsxImportSource @emotion/react */

import { useContext, useEffect, useRef } from 'react';
```

This will fix the tab styles on the state/tribe page.